scripts/manifests/firefox.json

```diff
@@ -53,7 +60,8 @@
 
   "permissions": [
     "storage",
-    "activeTab"
+    "activeTab",
+    "tabs"
```

Comment:
Not a full review yet, but can't we make that permission optional?

Technically, we can, at least: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_permissions

Reply:
Yes, we can. The reason I made it required is this it is needed to send any updated options to the content scripts in each tab. Without this permission, the use would have to reload all their tabs manually for the changes to take effect.

I changed it to optional in https://github.com/rugk/awesome-emoji-picker/pull/93/commits/16600e12b4c27cc34a8a9dd34f68df21ad34e864.